app/helpers/chapters_helper.rb

```diff
@@ -0,0 +1,42 @@
+module ChaptersHelper
```

Comment:
I would put this code in a presenter (like in Protected Planet) rather than a helper as the method within is only concerned with providing variables for the front end.